packages/user-operation-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.1.0]
```

Comment:
One more thing — if the change below isn't adding any features should we make this 12.0.1 instead of 12.1.0, following [SemVer](https://semver.org/)?

Reply:
Updated in [91a6881](https://github.com/MetaMask/core/pull/4460/commits/91a6881991b2079de6fa5f0e10446c434ddb17cd) and [2cf1e08](https://github.com/MetaMask/core/pull/4460/commits/2cf1e08601366c8262a2f8e0343140e77774a38d)